src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -27,7 +31,7 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
 
   def this() = this(ActorSystem("WorkflowSubmissionSpec"))
   val mockGoogleServicesDAO: MockGoogleServicesDAO = new MockGoogleServicesDAO("test")
-  val mockServer = RemoteServicesMockServer()
+  var mockServer = RemoteServicesMockServer()
```

Comment:
I don't like making this a var. Instead make the mock server a constructor parameter to TestWorkflowSubmission.